apps/ui/src/components/EditorChoices.vue

```diff
@@ -0,0 +1,69 @@
+<script setup lang="ts">
+import Draggable from 'vuedraggable';
+import { Draft } from '@/types';
+
+const proposal = defineModel<Draft>({ required: true });
+
+defineProps<{
+  definition: any;
+}>();
+</script>
+
+<template>
+  <div class="s-base mb-5">
+    <h4 class="eyebrow mb-2.5">Choices</h4>
+    <div class="flex flex-col gap-[10px]">
+      <Draggable
+        v-model="proposal.choices"
+        v-bind="{ animation: 200 }"
+        handle=".handle"
+        class="flex flex-col gap-[10px]"
+        item-key="index"
+      >
+        <template #item="{ index }">
+          <div>
+            <div class="flex items-center rounded-lg bg-skin-border h-[40px] gap-[12px] pl-2.5">
+              <div
+                class="text-skin-text"
+                :class="{
+                  'handle cursor-grab': proposal.type !== 'basic',
+                  'cursor-not-allowed': proposal.type === 'basic'
+                }"
+              >
+                <ICDrag />
```

Comment:
```suggestion
                <IC-drag />
```

We use this naming convention for icons.